core/src/validator.rs

```diff
@@ -2797,8 +2816,10 @@ mod tests {
                 &cluster_info,
                 rpc_override_health_check,
                 &start_progress,
-            ),
-            Err(ValidatorError::BadExpectedBankHash)
+            )
+            .unwrap_err()
+            .to_string(),
+            "Bad expected bank hash"
```

Comment:
Can just matches!() on the validator error `Err(ValidatorError::BadExpectedBankHash)`

Reply:
Done.